tidb-storage.md

```diff
@@ -12,68 +11,68 @@ This document introduces some design ideas and key concepts of TiKV.
 
-Note that the KV storage model for TiKV described in this article has nothing to do with Table in SQL. This article does not discuss any concepts in SQL and focuses on how to implement a high-performance, high-reliability, distributed Key-Value storage such as TiKV.
+Note that the KV storage model for TiKV described in this document has nothing to do with tables in SQL. This document does not discuss any concepts related to SQL and focuses on how to implement a high-performance, high-reliability, distributed Key-Value storage such as TiKV.
 
 ## Local storage (RocksDB)
 
-For any persistent storage engine, data will eventually be saved on disk, and TiKV is no exception. But TiKV did not choose to write data directly to the disk, but stored the data in RocksDB, which is responsible for the data landing. The reason for this choice is that there is a lot of work to be done to develop a stand-alone storage engine, especially to make a high-performance stand-alone engine that requires careful optimization,
-and RocksDB is an excellent stand-alone storage engine open source by Facebook, it can meet various requirements of single engine for TikV. Here, we can simply consider RocksDB as a single persistent Key-Value Map.
+For any persistent storage engine, data is eventually saved on disk, and TiKV is no exception. TiKV does not write data directly on the disk, but stores data in RocksDB, which is responsible for the data storage. The reason is that it costs a lot to develop a standalone storage engine, especially a high-performance standalone engine that requires careful optimization,
+RocksDB is an excellent standalone storage engine open-sourced by Facebook. This engine can meet various requirements of TiKV for a single engine. Here, you can simply consider RocksDB as a single persistent Key-Value Map.
 
 ## Raft protocol
 
-Next, the implementation of TiKV faces a more difficult thing: how to ensure that the single failure of the case, data is not lost, error-free?
+What's more, the implementation of TiKV faces a more difficult thing: how to secure data safety in case a single machine fails?
 
-In simple terms, you need to find a way to replicate data to multiple machines, so that one machine doesn't work, and the replicas on other machines do; in complex terms, you need the data replication scheme to be reliable and efficient, and be able to handle the failure of a replica. TiKV chose the Raft algorithm.
+A simple way is to replicate data to multiple machines, so that even if one machine fails, the replicas on other machines are still available. To put it difficult, you need a data replication scheme that is reliable, efficient, and able to handle the situation of a failed replica. All of these are made possible by the Raft algorithm.
```

Comment:
```suggestion
A simple way is to replicate data to multiple machines, so that even if one machine fails, the replicas on other machines are still available. In other words, you need a data replication scheme that is reliable, efficient, and able to handle the situation of a failed replica. All of these are made possible by the Raft algorithm.
```